src/components/measureList/MeasureList.tsx

```diff
@@ -255,15 +255,15 @@ export default function MeasureList(props: {
 
   const exportMeasure = async () => {
     try {
-      const blobResponse = await measureServiceApi.getMeasureExport(
-        targetMeasure?.current?.id
+      const { ecqmTitle, model, version } = targetMeasure?.current ?? {};
+      const url = window.URL.createObjectURL(
+        await measureServiceApi?.getMeasureExport(targetMeasure.current?.id)
       );
-      const url = window.URL.createObjectURL(blobResponse);
       const link = document.createElement("a");
       link.href = url;
       link.setAttribute(
         "download",
-        `${targetMeasure.current?.ecqmTitle}-v${targetMeasure.current?.version}-${targetMeasure.current?.model}.zip`
+        `${ecqmTitle}-v${version}-${getModelFamily(model)}.zip`
       );
       document.body.appendChild(link);
```

Comment:
not part of your task but this seems not so correct. it will keep appending the anchors to the body. we'll have to remove the anchor after the download

Reply:
I have implemented that and pushed those changes